initializr-generator/src/test/groovy/io/spring/initializr/test/metadata/InitializrMetadataTestBuilder.groovy

```diff
@@ -152,6 +152,13 @@ class InitializrMetadataTestBuilder {
 		this
 	}
 
+	InitializrMetadataTestBuilder addCustomParentPomGAV(String customGAV) {
```

Comment:
`setParentPom(groupId, artifactId, version, addSpringBootBom)` (there is only one parent).